README.md

```diff
@@ -12,6 +13,8 @@ You want your Arduino library to be automatically built and tested every time so
 
 `arduino_ci` is a cross-platform build/test system, consisting of a Ruby gem and a series of C++ mocks.  It enables tests to be run both locally and as part of a CI service like GitHub Actions, TravisCI, Appveyor, etc.  Any OS that can run the Arduino IDE can run `arduino_ci`.
 
+> Note: for testing GitHub pull requests, an [Arduino CI GitHub Action](https://github.com/marketplace/actions/arduino_ci) is available for your convenience.  This method of running `arduino_ci` is driven by Docker, which may also serve your local testing needs (as it does not require a ruby environment to be installed)
```

Comment:
I think the official term is "events":
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows
the term "actions" already gets a lot of use when talking about this subject (e.g., "`Arduino-CI/action` is a GitHub Actions action"), so it's best to avoid unnecessary use of the word in this context.

Reply:
Yes, it was awkward writing that sentence; "events" is better.

Reply:
Good catch, fixing this